code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -179,11 +180,24 @@
 /datum/species/golem/silver/on_species_loss(mob/living/carbon/C)
 	REMOVE_TRAIT(C, TRAIT_HOLY, SPECIES_TRAIT)
 	..()
+// Softer and faster, but conductive
+/datum/species/golem/copper
+	name = "Copper Golem"
+	id = "copper_golem"
+	fixed_mut_color = "#d95802"
+	speedmod = 1.5

```

Comment:
This is actually slower than about half of the golems. Lower speedmod is faster, higher is slower.